build.gradle

```diff
@@ -116,6 +116,22 @@ dependencies {
   testImplementation 'io.fabric8:kubernetes-server-mock:6.7.2', {
     // self managed to avoid conflicts
     exclude group: "org.slf4j"
+    exclude group: "junit"
+  }
+  testImplementation "junit:junit:4.13.1"
```

Comment:
Why do we need this JUnit 4 dependency?

Reply:
I tried to drop it completely, but the server mock fails then.